### PR TITLE
Release/0.2.1

### PR DESCRIPTION
**Pull request by kaywolter**

### CODE REVIEW

File: loghero/LogHeroAPISettings.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        );
+        file_put_contents($this->apiDevSettingsFile, json_encode($jsonData));
+    }
+
+    private function initializeFromDevSettingsFile() {
+        if (!file_exists($this->apiDevSettingsFile)) {
+            return;
+        }
+        $jsonString = file_get_contents($this->apiDevSettingsFile);
+        chmod($this->apiDevSettingsFile, 0666);
```
Comment (mhergerdt) [inline loghero/LogHeroAPISettings.php]: Is this chmod needed? Seems like it has nothing to do with this method?

File: loghero/LogHeroAPISettings.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        if (!$json) {
+            return;
+        }
+        if (array_key_exists('apiLogPackageEndpoint', $json)) {
+            $this->apiLogPackageEndpoint = $json['apiLogPackageEndpoint'];
+        }
+    }
+
+    private function devSettingsFileNeedsUpdate() {
+        return $this->hasCustomSettings() or file_exists($this->apiDevSettingsFile);
```
Comment (mhergerdt) [inline loghero/LogHeroAPISettings.php]: In this case "or" is fine but be aware if you have a statement like this:

File: loghero/LogHeroAPISettings.php (hunk 1 above)
Comment (kaywolter) [reply loghero/LogHeroAPISettings.php]: Good catch. This chmod belongs into the function above. Done.

File: loghero/LogHeroAPISettings.php (hunk 2 above)
Comment (kaywolter) [reply loghero/LogHeroAPISettings.php]: Thanks, this was not intended. In Python this is the equivalent for "||".

File: loghero/readme.txt [hunk 3]
```diff
@@ -101,11 +101,17 @@ You'll find answers to many of your questions on [log-hero.com](https://log-hero
 
 == Changelog ==
 
-First Version of this Plugin
+=== [0.2.1] ===
```
[the remark this answers is not in the document]
Comment (kaywolter) [reply loghero/readme.txt]: This is the format we have to follow for the Wordpress plugin directory to show the changelog on the plugin site: https://wordpress.org/plugins/loghero/#developers. I agree that keeping it in a separate file is cleaner.

**Pull request merged**